dask/array/tests/test_creation.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from numpy.testing import assert_array_almost_equal
+
+from dask.array import creation
+
+
+def test_linspace():
+    darr = creation.linspace(6, 49, 5)
+    nparr = np.linspace(6, 49)
+    assert_array_almost_equal(np.array(darr), nparr)
+
+
+def test_arange():
+    darr = creation.arange(77, blocksize=13)
+    nparr = np.arange(77)
+    assert_array_almost_equal(np.array(darr), nparr)
+
+    darr = creation.arange(2, 13, blocksize=5)
+    nparr = np.arange(2, 13)
+    assert_array_almost_equal(np.array(darr), nparr)
+
+    darr = creation.arange(4, 21, 9, blocksize=13)
+    nparr = np.arange(4, 21, 9)
+    assert_array_almost_equal(np.array(darr), nparr)
```

Comment:
`numpy.arange` doesn't support non integer stepsizes so we probably should not. It recommends using linspace for those cases. 

 http://docs.scipy.org/doc/numpy/reference/generated/numpy.arange.html#numpy-arange